packages/iris-grid/src/IrisGridModelUpdater.jsx

```diff
@@ -97,6 +98,9 @@ const IrisGridModelUpdater = React.memo(
     useEffect(() => {
       model.pendingDataMap = pendingDataMap;
     }, [model, pendingDataMap]);
+    useEffect(() => {
+      model.contextFrozenColumns = contextFrozenColumns;
```

Comment:
In IrisGridModel, you called it `uiFrozenColumns`. This should be `model.uiFrozenColumns`